polkadot/node/core/pvf/src/testing.rs

```diff
@@ -62,6 +62,29 @@ pub fn validate_candidate(
 pub fn get_and_check_worker_paths() -> (PathBuf, PathBuf) {
 	// Only needs to be called once for the current process.
 	static WORKER_PATHS: OnceLock<Mutex<(PathBuf, PathBuf)>> = OnceLock::new();
+
+	fn maybe_build_workers(build_prep: bool, build_exec: bool) {
+		let mut build_args = vec!["build", "--package=polkadot"];
+		if build_prep {
+			build_args.push("--bin=polkadot-prepare-worker");
+		}
+		if build_exec {
+			build_args.push("--bin=polkadot-execute-worker");
+		}
+
+		if build_prep || build_exec {
+			eprintln!("Building workers...");
+
+			let dir = env!("CARGO_MANIFEST_DIR");
+			std::process::Command::new("cargo")
```

Comment:
Can we pass `SKIP_WASM_BUILD=1`? I don't think we need the runtimes to be built for the workers.